src/optimistic-oracle-v2/implementation/ManagedOptimisticOracleV2.sol

```diff
@@ -40,13 +42,26 @@ contract ManagedOptimisticOracleV2 is
     ManagedOptimisticOracleV2Events,
     OptimisticOracleV2,
     AccessControlDefaultAdminRulesUpgradeable,
+    Ownable2StepUpgradeable,
```

Comment:
Why not just use a role for this rather than trying to use Ownable and AccessControl in the same contract? Can't we do all the same stuff with AccessControl?

Reply:
Yes, AccessControl allows creating another role for contract upgrades, but then the default admin (the one who also adds/removes request managers) would be able to add/remove contract upgraders, which kind of defeats the purpose trying to segregate contract config admin from upgrade admin

Reply:
Though one solution could be to build on top of internal `_setRoleAdmin` in the AccessControl by setting the role admin for the upgrader role to itself. I could try implement this tomorrow.

Reply:
Fixed, so now all roles are managed in AccessControl:
- `DEFAULT_ADMIN_ROLE` - single super admin with 2-step transfer protection, can upgrade the contract
- `REGULAR_ADMIN` - role to set contract parameters, managed by `DEFAULT_ADMIN_ROLE`
- `REQUEST_MANAGER` - role for managing per request settings, managed by `REGULAR_ADMIN`